articles/automation/automation-windows-hrw-install.md

```diff
@@ -64,7 +64,7 @@ Perform the following steps to automate the installation and configuration of th
 3. From the PowerShell command-line shell, navigate to the folder, which contains the script you downloaded and execute it changing the values for parameters *-AutomationAccountName*, *-ResourceGroupName*, *-HybridGroupName*, *-SubscriptionId*, and *-WorkspaceName*.
 
      > [!NOTE] 
-     > You are prompted to authenticate with Azure after you execute the script.  You **must** sign in with an account that is a member of the Subscription Admins role and co-administrator of the subscription.  
+     > You are prompted to authenticate with Azure after you execute the script. This requires JavaScript to be enabled on your browser. As well, you **must** sign in with an account that is a member of the Subscription Admins role and co-administrator of the subscription.  
```

Comment:
Can you change your addition to the following, the permissions are the main point of the note here, I do not mind to add the javascript information but it is second to the permissions needed.

```
You are prompted to authenticate with Azure after you execute the script. You **must** sign in with an account that is a member of the Subscription Admins role and co-administrator of the subscription. This prompt does require JavaScript to be enabled on your browser. 
```